inflate.c

```diff
@@ -237,24 +237,24 @@ int Z_INTERNAL inflate_ensure_window(struct inflate_state *state) {
    output will fall in the output data, making match copies simpler and faster.
    The advantage may be dependent on the size of the processor's data caches.
  */
-static int32_t updatewindow(PREFIX3(stream) *strm, const uint8_t *end, uint32_t copy) {
+static int32_t updatewindow(PREFIX3(stream) *strm, const uint8_t *end, uint32_t len, int copy) {
```

Comment:
I think perhaps the `copy` variable should be renamed `checksum`. I think that's right.. it is turning on and off checksum with copy.. the else statements still copy..

Reply:
Ah yes, you are correct.